pkg/api/types.go

```diff
@@ -21,7 +21,7 @@ const (
 
 // invalidFilenameCharacters contains a list of character we consider malicious
 // when injecting the directories into containers.
-const invalidFilenameCharacters = `\:;*?"<>|%#$!+{}&[],"'` + "`"
+const invalidFilenameCharacters = `\;*?"<>|%#$!+{}&[],"'` + "`"
```

Comment:
Why are we removing `:`?

If we are removing this to accept paths like `C:\Users\name\file.txt`, however, seems that we're then being less restrictive about file names containing `:`?!

And what about `\`, wouldn't it be the same case as `:`?